File: Maths/02-FizzBuzz/solutions/hunj-fizzbuzz.swift
func Fizzbuzz(number: Int) -> String {
	if number % 3 == 0 && number % 5 == 0 {
		return "FizzBuzz"
	}
	else if number % 5 == 0 {
		return "Buzz"
	}
	else if number % 3 == 0 {
		return "Fizz"
	}
	else {
		return String(number)
	}
}

for i in 1...100 {
    println( Fizzbuzz(i) )
}

File: Maths/02-FizzBuzz/solutions/hunj-fizzbuzz.rb
def fizzbuzz(number)
  if number % 5 == 0 && number % 3 == 0
    puts 'FizzBuzz'
  elsif number % 5 == 0
    puts 'Buzz'
  elsif number % 3 == 0
    puts 'Fizz'
  else
    puts number
  end
end

(1..100).each do |n|
  fizzbuzz n
end
